### PR TITLE
Form API response CSS classes

### PR DESCRIPTION
**Pull request by mthommes**
When you submit a subscription form through our PHP wrapper (Ajax), we include some JavaScript that updates the DOM with a result message (IE: "Contact added," "Contact updated," etc). I added a class to the message div in the DOM based on success or failure with the Ajax call. This way developers can style the inline response message based on success or failure.

### CODE REVIEW

File: includes/Form.class.php [hunk 1]
```diff
@@ -121,6 +121,9 @@ function embed($params) {
 			},
 			success: function(data) {
 				\$j('#form_result_message').html(data.message);
+				var result_class = (data.success) ? 'form_result_success' : 'form_result_error';
+				\$j('#form_result_message').removeClass();
```
Comment (srdjevic) [inline includes/Form.class.php]: we sure we want to remove _all classes_?

instead we can do $J(selector).removeClass("form_result_success form_result_error").addClass(result_class); // does the same, but doesn't remove any extra classes

Comment (mthommes) [inline includes/Form.class.php]: Good idea. Updated. :+1:

File: includes/Form.class.php [hunk 2]
```diff
@@ -121,6 +121,9 @@ function embed($params) {
 			},
 			success: function(data) {
 				\$j('#form_result_message').html(data.message);
+				var result_class = (data.success) ? 'form_result_success' : 'form_result_error';
+				\$j('#form_result_message').removeClass('form_result_success form_result_error').addClass(result_class);
+				\$j('#form_result_message').addClass(result_class);
```
Comment (srdjevic) [inline includes/Form.class.php]: can take this line out then

Comment (mthommes) [inline includes/Form.class.php]: Ugh, I missed that - good catch. Updated.

**Pull request merged**